core/sequencer/server.go

```diff
@@ -63,6 +63,9 @@ var (
 	unappliedRevisions monitoring.Gauge
 )
 
+// zero is the time to use for the beginning edge of new watermarks.
+var zero = time.Unix(0, 0)
```

Comment:
Can this be a const?

Reply:
golang doesn't support constants for structs.